properties.go

```diff
@@ -94,6 +94,15 @@ const (
 	// multicast on startup.
 	EnvVarMulticastEnabled = "SMUDGE_MULTICAST_ENABLED"
 
+	// EnvVarMulticastAnnounceInterval is the name of the environment variable
+	// that describes whether Smudge will attempt to re-announce its presence
+	// via multicast every X seconds.
+	EnvVarMulticastAnnounceInterval = "SMUDGE_MULTICAST_ANNOUNCE_INTERVAL"
+
+	// DefaultMulticastAnnounceInterval is the default value for whether Smudge
+	// will re-announce its presence via multicast
+	DefaultMulticastAnnounceInterval = 10
```

Comment:
For consistency and clarity, this should include the units used as well: `DefaultMulticastAnnounceSeconds` or `DefaultMulticastAnnounceMillis` probably.